x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -81,6 +82,7 @@ export const CreateCaseFlyout = React.memo<CreateCaseFlyoutProps>(
         <GlobalStyle />
         <StyledFlyout
           onClose={onClose}
+          tour-step="create-case-flyout"
```

Comment:
These are labels for my tour step anchor. Previously I was using `data-test-subj="create-case-flyout"` but it makes me nervous to call on a test ref in production.